ansible_navigator/actions/inventory.py

```diff
@@ -435,12 +435,63 @@ def _build_inventory_list(self) -> None:
         self._set_inventories_mtime()
         return
 
-    def _collect_inventory_details(
+    def _collect_inventory_details_interactive(
+        self,
+        kwargs: Dict[str, Any],
+    ) -> None:
+        self._runner = InventoryRunner(**kwargs)
+        inventory_output, inventory_err = self._runner.fetch_inventory("list", self._inventories)
+        if inventory_output:
+            parts = inventory_output.split("{", 1)
+            if inventory_err:
+                inventory_err = parts[0] + inventory_err
+            else:
+                inventory_err = parts[0]
+
+            if len(parts) == 2:
+                inventory_output = "{" + parts[1]
+            else:
+                inventory_output = ""
+        warn_msg = ["Errors were encountered while gathering the inventory:"]
+        warn_msg += inventory_err.splitlines()
+        self._logger.error(" ".join(warn_msg))
+        if "ERROR!" in inventory_err or "Error" in inventory_err:
+            warning = warning_notification(warn_msg)
+            self._interaction.ui.show(warning)
+        else:
+            self._extract_inventory(inventory_output)
+
+    def _collect_inventory_details_automated(
```

Comment:
Could you submit this refactoring as a separate PR so that it'd be easier to review this one with a smaller diff?

Reply:
> Could you submit this refactoring as a separate PR so that it'd be easier to review this one with a smaller diff?

https://github.com/ansible/ansible-navigator/pull/563